pkg/kubelet/metrics/metrics.go

```diff
@@ -938,6 +942,18 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAOperationsDuration tracks the duration of the DRA PrepareResources and UnprepareResources requests.
+	DRAOperationsDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      DRASubsystem,
+			Name:           DRAOperationsDurationKey,
+			Help:           "Duration in seconds of the DRA operations",
```

Comment:
The help text doesn't add additional information, it just paraphrases the metric name. Perhaps explain that these "operations" are per-pod, not per-driver?